packages/docusaurus/docs/advanced/04-technical/sherlock.md

```diff
@@ -0,0 +1,147 @@
+---
+category: advanced
+slug: /advanced/sherlock
+title: "Sherlock"
+description: An overview of Yarn's GitHub Actions issue toolkit that helps users provide minimal reproductions.
+hidden: true
```

Comment:
I'm not sure if this does work on newer Docusarus-based website but I don't think it should be a dealbreak.